packages/types/types/ts3.6/index.d.ts

```diff
@@ -0,0 +1,173 @@
+import { Action } from 'redux'
+
+export type Saga<Args extends any[] = any[]> = (...args: Args) => IterableIterator<any>
+
+/**
+ * Annotate return type of generators with `SagaIterator` to get strict
+ * type-checking of yielded effects.
+ */
+export type SagaIterator = IterableIterator<StrictEffect>
```

Comment:
Did we lose `export type SagaIterator<RT = any> = Iterator<StrictEffect, RT, any>` here?

Reply:
Yeah, sorry, I messed up my git checkouts apparently. I work on this at night, after work, which is very prone to mistakes 😅. I wanted to keep backwards compatibility with TS<3.6. I'll restore TS3.6 typings later, probably tommorow. The commit should still exist in the tree.

I'll probably also try to drop typings-tester in favor of dtslint to have proper tests for each types versions sometime.